src/components/select-field/constants.js

```diff
@@ -0,0 +1,4 @@
+// @flow
+const CLEAR: '__clear__' = '__clear__'; // value of clear option used in select-field components
```

Comment:
Using a magic constant like this will help us avoid any collisions where an enduser might have an option with a value of "clear". It is highly unlikely that they will have the same constant as us now.